pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -137,7 +141,17 @@ func (p *cadvisorStatsProvider) ListPodStats() ([]statsapi.PodStats, error) {
 			copy(ephemeralStats, vstats.EphemeralVolumes)
 			podStats.VolumeStats = append(append([]statsapi.VolumeStats{}, vstats.EphemeralVolumes...), vstats.PersistentVolumes...)
 		}
-		podStats.EphemeralStorage = calcEphemeralStorage(podStats.Containers, ephemeralStats, &rootFsInfo, nil, false)
+
+		logStats, err := p.hostStatsProvider.getPodLogStats(podStats.PodRef.Namespace, podStats.PodRef.Name, podUID, &rootFsInfo)
+		if err != nil {
+			klog.Errorf("Unable to fetch pod log stats: %v", err)
```

Comment:
I worry about these two klog.Errorf's here... They probably need `continues` since logStats and etcHostsStats might be nil. Thoughts?

Reply:
Both `logStats` and `etcHostsStats` are only passed to the helper function `calcEphemeralStorage()`. That function does a nil check before using, so I think we should be safe. I think it would be good to carry on in the function if these errors are encountered. My thinking is that if one part of the metric can't be computed, we still want to do the best that we can with the other parts.

Reply:
I am fine with the present form.